charts/buildbuddy/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: BuildBuddy Open Source
 name: buildbuddy
 version: 0.0.31 # Chart version
```

Comment:
Could you bump the chart version and leave the app version at 2.1.0?

Reply:
Ah, sorry! I totally blanked and just saw a semver and incremented it 😅 fixed

Reply:
I've been there!